crates/web-sys/webidls/enabled/CanvasCaptureMediaStream.webidl

```diff
@@ -10,7 +10,7 @@
  * W3C liability, trademark and document use rules apply.
  */
 
-[Pref="canvas.capturestream.enabled"]
+[Deprecated]
```

Comment:
If you want this to actually be marked deprecated on the Rust side, you can use `[RustDeprecated = "..."]`.

`[Deprecated]` is another Firefox holdover that the WebIDL generator ignores.

Reply:
@Liamolucko do you think it makes sense to mark a non-standard interface like this as deprecated on the Rust side? Is this consistent with other non-standard interfaces that we support?

Reply:
I don't think there are any other non-standard interfaces in `web-sys` that we 'support'. There might some there by mistake, which wouldn't be deprecated (nothing is currently deprecated in `web-sys`), but that's not by any conscious choice so I wouldn't really take it as a precedent.

It feels a bit weird not to have any indication that this API is non-standard, but deprecating it would just be an inconvenience to anyone who actually needed to use it. So, I'm leaning towards not deprecating it.

Reply:
I agree. On one hand, I think it would be nice to indicate to developers using `web-sys` that they are using an interface that is not (or should not be) supported by browsers, but on the other hand, I can also see this as annoyance. In the long run, perhaps a general policy or approach could be adopted for such interfaces but that is certainly beyond the scope of this PR.